lib/octokit/client/integrations.rb

```diff
@@ -0,0 +1,77 @@
+module Octokit
+  class Client
+
+    # Methods for the Integrations API
+    module Integrations
+
+      # Find all installations that belong to an Integration
+      #
+      # @param options [Hash] An customizable set of options
+      #
+      # @see https://developer.github.com/v3/integrations/#find-installations
+      #
+      # @return [Array<Sawyer::Resource>] A list of installations
+      def find_integration_installations(options = {})
+        options = ensure_api_media_type(:integrations, options)
+        paginate "/integration/installations", options
+      end
+      alias :find_installations :find_integration_installations
+
+      # Create a new installation token
+      #
+      # @param installation [Integer] The id of a a GitHub Integration Installation
+      # @param options [Hash] An customizable set of options
+      #
+      # @see https://developer.github.com/v3/integrations/#find-installations
+      #
+      # @return [<Sawyer::Resource>] An installation token
+      def create_integration_installation_access_token(installation, options = {})
+        options = ensure_api_media_type(:integrations, options)
```

Comment:
As a pattern across the library, we avoid mutating the parameters. In other methods we usually use `opts` as the variable for options after mutating them. In this case it would unlikely cause issues, but would bring everything into parity to follow the same pattern.